### PR TITLE
Added :verbose-output subroutine, made aliases update more functional

### PR DESCRIPTION
**Pull request by daxgames**

### CODE REVIEW

File: vendor/init.bat [hunk 1]
```diff
@@ -80,7 +83,7 @@ if exist "%CMDER_ROOT%\vendor\git-for-windows" (
 :: Add git to the path
 if defined GIT_INSTALL_ROOT (
     rem add the unix commands at the end to not shadow windows commands like more
-    echo Enhancing PATH with unix commands from git in "%GIT_INSTALL_ROOT%\usr\bin"
+    call :verbose-output Enhancing PATH with unix commands from git in "%GIT_INSTALL_ROOT%\usr\bin"
```
Comment (janschulz) [inline vendor/init.bat]: I actually like this output in the console...

File: vendor/init.bat [hunk 2]
```diff
@@ -165,3 +182,12 @@ if exist "%CMDER_ROOT%\config\user-profile.cmd" (
     echo.
     ) > "%CMDER_ROOT%\config\user-profile.cmd"
 )
+
+exit /b
+
+::
+:: sub-routines below here
+::
+:verbose-output
```
Comment (janschulz) [inline vendor/init.bat]: `LOG_DEBUG` (and `LOG_INFO`)?

File: vendor/init.bat [hunk 3]
```diff
@@ -6,6 +6,9 @@
 :: !!! THIS FILE IS OVERWRITTEN WHEN CMDER IS UPDATED
 :: !!! Use "%CMDER_ROOT%\config\user-profile.cmd" to add your own startup commands
 
+:: Set to > 0 for verbose output to aid in debugging.
+set verbose-output=0
```
Comment (janschulz) [inline vendor/init.bat]: IMO this should be setable from outside the init.bat...

File: vendor/init.bat (hunk 2 above)
Comment (daxgames) [inline vendor/init.bat]: I don't understand the comment.  Are you saying there should be a log_debug and a log_info subrouteen also?

File: vendor/init.bat [hunk 4]
```diff
@@ -6,6 +6,9 @@
 :: !!! THIS FILE IS OVERWRITTEN WHEN CMDER IS UPDATED
 :: !!! Use "%CMDER_ROOT%\config\user-profile.cmd" to add your own startup commands
 
+:: Set to > 0 for verbose output to aid in debugging.
+if not defined verbose-output ( set verbose-output=0 )
+
```
Comment (daxgames) [inline vendor/init.bat]: Done.

Add 'set verbose-output=1 && ' before 'cmd /k' in cmder task definition.

File: vendor/init.bat (hunk 3 above)
Comment (daxgames) [inline vendor/init.bat]: Done.

Add 'set verbose-output=1 && ' before 'cmd /k' in cmder task definition.

File: vendor/init.bat (hunk 4 above)
Comment (daxgames) [inline vendor/init.bat]: lLke this: cmd /k "%ConEmuDir%\..\init.bat"  -new_console:d:%USERPROFILE%

Comment (daxgames) [inline vendor/init.bat]: Like this:
set verbose-output=1 && cmd /k "%ConEmuDir%\..\init.bat"  -new_console:d:%USERPROFILE%

File: vendor/init.bat (hunk 1 above)
Comment (Stanzilla) [inline vendor/init.bat]: I don't :smile:

File: vendor/init.bat (hunk 2 above)
Comment (janschulz) [inline vendor/init.bat]: No, instead of `verbose-output`

File: vendor/init.bat (hunk 4 above)
Comment (janschulz) [inline vendor/init.bat]: Nice, thanks!

File: vendor/init.bat (hunk 1 above)
Comment (daxgames) [inline vendor/init.bat]: Yeah me either that's why I did the above.

File: vendor/init.bat (hunk 2 above)
Comment (daxgames) [inline vendor/init.bat]: I don't care what it's called I just did like a way to easily enable/disable some additional info that could help in troubleshooting.

Comment (daxgames) [inline vendor/init.bat]: :-)

**Pull request merged**